src/lib.rs

```diff
@@ -109,7 +109,7 @@ pub fn initialize(_argc: *const isize, _argv: *const *const *const u8) -> isize
 /// use libfuzzer_sys::fuzz_target;
 ///
 /// // Note: `|input|` is short for `|input: &[u8]|`.
-/// fuzz_target!(|input| {
+/// fuzz_target!(|input: &[u8]| {
```

Comment:
This should not be necessary because, as the `Note:` comment just above implies, omitting the type is equivalent to specifying it as `&[u8]` and there are a bunch of fuzz targets out there that omit the type already, and they should continue to work.

Reply:
That is, instead of changing this doc example, the macro should be fixed to allow this pattern again.

Reply:
Rushed the fix; I have now extended the macro and also updated the doc accordingly, showing a call with init.